bittide/tests/Tests/Shared.hs

```diff
@@ -266,3 +268,32 @@ fromListWithResetAndEnable inp =
       | rst       = inpAsSignal
       | ena       = xs
       | otherwise = x :- xs
+
+-- | A simple simulation-only driver that receives a list of Wishbone master requests
+-- alongside the slave bus and drives the master bus. The driver presents the next
+-- request after the current request has been terminated by `acknowledge`, `retry` or `err`.
+wishboneSimDriver ::
```

Comment:
Couldn't this use `driveStandard` from `clash-protocols`?

Reply:
I tried it and indeed. I'll put this in draft for now and close this PR once all instances have been replaced with `driverStandard`.